models/characters/character.gd

```diff
@@ -92,6 +94,7 @@ func update_character():
 
 	if scene and respath:
 		var instance = scene.instantiate()
+		self.add_child(instance)
```

Comment:
this taints `self` with a second Armature and AnimationPlayer, does it still work if we instance.queue_free() after reparenting?

Reply:
Yes indeed! I just tested it and it works fine. I am going to edit the pr